### DISCUSSION TITLE
Which extensions and protocols do you support?

### DISCUSSION THREAD

**Discussion by alicansag** [xmppjs/xmpp.js#991] [Q&A]
Hello,
I have just found this repository and couldn't find any information on which XMPP extensions and protocols you have implemented? For example, see [jsxc's list of supported XEPs](https://www.jsxc.org/features.html). Do you have such a list?

Also this repo is not listed in [xmpp.org's list](https://xmpp.org/software/), therefore I am assuming that you are not an offical implementation yet, am I right?

Thanks in advance.

**Comment by alicansag** [comment]
Apologies, I have found this repo in the "libraries" section in xmpp's website.

**Comment by xlrshubham1** [comment]
I have just found this repository and couldn't find any information on which XMPP extensions and protocols you have implemented? For example, see [jsxc's list of supported XEPs](https://www.jsxc.org/features.html). Do you have such a list?
